Fix bug: Customer import deletes exiting customer entity Fields

jalogut opened the pull request:
Importing a import file to update customer data, results in `entity fields` being removed if the columns are not present on the imported file.

That is the case for all fields defined here:

```
protected $customerFields = [
        CustomerInterface::GROUP_ID,
        CustomerInterface::STORE_ID,
        CustomerInterface::UPDATED_AT,
        CustomerInterface::CREATED_AT,
        CustomerInterface::CREATED_IN,
        CustomerInterface::PREFIX,
        CustomerInterface::FIRSTNAME,
        CustomerInterface::MIDDLENAME,
        CustomerInterface::LASTNAME,
        CustomerInterface::SUFFIX,
        CustomerInterface::DOB,
        'password_hash',
        CustomerInterface::TAXVAT,
        CustomerInterface::CONFIRMATION,
        CustomerInterface::GENDER,
        'rp_token',
        'rp_token_created_at',
        'failures_num',
        'first_failure',
        'lock_expires',
    ];
```

Importing a customer should result on adding or editing new data but it should keep previous information if the column is not present on imported file.

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

jalogut committed:
Fix bug that deletes exiting customer entity Fields, if the columns are not present on Import file

osrecio commented on app/code/Magento/CustomerImportExport/Model/Import/Customer.php:
```diff
@@ -270,13 +270,23 @@ protected function _saveCustomerEntities(array $entitiesToCreate, array $entitie
+        $customerFieldsToUpdate = array_filter($this->customerFields, function ($field) use ($columnsToUpdate) {
+            return in_array($field, $columnsToUpdate);
+        });
+        return $customerFieldsToUpdate:
```
Change `:` to `;` Please (Line 287)

jalogut replied:
ups! I edited the commit. Thanks for noticing.

dmanners commented on app/code/Magento/CustomerImportExport/Model/Import/Customer.php:
```diff
@@ -270,13 +270,23 @@ protected function _saveCustomerEntities(array $entitiesToCreate, array $entitie
         return $this;
     }
 
+    private function getCustomerEntityFieldsToUpdate(array $entitiesToUpdate): array
```
Could you maybe a docblock to this method with a short explanation of the bug this method fixes. Is there actually a github issue for this fix cause you could add a link to that here.

dmanners committed:
Filter the import to make sure we only update fields present in the import file.
 - https://github.com/magento/magento2/pull/11968
 - Importing a import file to update customer data, results in entity fields being removed if the columns are not present on the imported file.
